src/Services/src/Threading/IThreadingService.cs

```diff
@@ -13,6 +14,8 @@ public interface IThreadingService
         Task ExecuteInUIThreadAsync(Action method);
         Task RemoveItemFromCollectionOnUiThreadAsync<T>(ObservableCollection<T> list, T item);
         Task StartBackgroundTask(Func<Task> method);
+        Task StartBackgroundTask(Func<Task> method, CancellationToken cancellationToken);
+        Task StartCancelableBackgroundTask(Func<string, CancellationToken, Task> method, string arg1, CancellationToken cancellationToken);
```

Comment:
Nonissue, but Cancelable looked odd to me (due to Cancellation being correct spelling) so I looked it up b/c why not:
```
Cancelable or cancellable
The word cancelable (or cancellable) is the adjective form of cancel. As you might guess, “cancelable” is the preferred spelling in America because it was one l, while “cancellable” is preferred outside of the U.S. 
```